client/pages/signup.jsx

```diff
@@ -1,132 +1,171 @@
-              <label htmlFor="password" className="float-left required">
-                Password
-              </label>
-              <input
-                type="password"
-                className="form-control"
-                id="password"
-                placeholder="Enter your password"
-                value={password}
-                onChange={event => {
-                  setPassword(event.target.value);
-                }}
-                required
-              />
-              <small
-                id="emailHelp"
-                className="form-text text-muted float-left mb-3"
-              >
-                Must contain at least 6 alphanumeric characters.
-              </small>
-              <br />
-            </div>
-
-            <button
-              type="submit"
-              className="btn btn-primary float-left"
-              onClick={async event => {
-                setErrorMessage("");
-                setSuccessMessage("");
-                event.preventDefault();
-                const { data } = await client.mutate({
-                  mutation: addUserMutation,
-                  variables: { email, password }
-                });
-                if (data.addUser.email === "empty") {
-                  setErrorMessage("You must enter a valid email address");
-                } else if (data.addUser.email === "taken") {
-                  setErrorMessage("That email address is already in use.");
-                } else {
-                  setSuccessMessage("Your account has been created!");
-                }
-                setEmail("");
-                setPassword("");
-              }}
-            >
-              Sign in
-            </button>
-          </form>
-          {errorMessage.length ? (
-            <p className="text-center text-danger">{errorMessage}</p>
-          ) : (
-            successMessage.length > 0 && (
-              <p className="text-center text-success">{successMessage}</p>
-            )
-          )}
-
-          <style>{`
-        .form-signin {
-          text-align: center;
-          width: 100%;
-          max-width: 400px;
-          margin: auto;
-          padding-top: 40px;
-          padding-bottom: 40px;
-        }
-
-        .form-signin .form-control {
-          padding: 10px;
-        }
-
-        .required:after{ 
-          content:'*'; 
-          color:red; 
-          padding-left:5px;
-        }
-      `}</style>
+          <div className="box-container">
+            <SignUpBox apolloClient={client} />
+          </div>
+          <style jsx>{`
+            .box-container {
+              text-align: center;
+              width: 100%;
+              max-width: 400px;
+              margin: auto;
+              padding-top: 40px;
+              padding-bottom: 40px;
+            }
+          `}</style>
         </Layout>
       )}
     </ApolloConsumer>
   );
 };
+
+export default Signup;
+
+// export const addUserMutation = gql`
```

Comment:
Please delete all unnecessary comments :bow:

Reply:
Oh, dear. That was embarrassing!